patterns/product-collections-featured-collection.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Title: Product Collections: Featured Collection
+ * Slug: woocommerce-blocks/product-collections-featured-collection
+ * Categories: WooCommerce
+ * Block Types: core/query/woocommerce/product-query
+ */
+?>
+
+<!-- wp:columns {"align":"wide","style":{"color":{"background":"#333333"},"spacing":{"padding":{"top":"1.3rem","right":"1.3rem","bottom":"1.3rem","left":"1.3rem"}}},"textColor":"white"} -->
+<div class="wp-block-columns alignwide has-white-color has-text-color has-background" style="background-color:#333333;padding-top:1.3rem;padding-right:1.3rem;padding-bottom:1.3rem;padding-left:1.3rem">
+	<!-- wp:column {"verticalAlignment":"center","width":"33.33%","style":{"spacing":{"padding":{"right":"var:preset|spacing|40","left":"var:preset|spacing|40"}}}} -->
+	<div class="wp-block-column is-vertically-aligned-center" style="padding-right:var(--wp--preset--spacing--40);padding-left:var(--wp--preset--spacing--40);flex-basis:33.33%">
+		<!-- wp:heading {"textAlign":"center","textColor":"background","fontSize":"x-large"} -->
+		<h2 class="wp-block-heading has-text-align-center has-background-color has-text-color has-x-large-font-size">This week’s popular products</h2>
```

Comment:
I don't have a strong preference, but I think we usually use this `'` apostrophe character:
```suggestion
		<h2 class="wp-block-heading has-text-align-center has-background-color has-text-color has-x-large-font-size">This week's popular products</h2>
```

Also, I guess we should make this string translatable, right?